### DISCUSSION TITLE
Any plans to add support of python3.11 and python3.12?

### DISCUSSION THREAD

**Discussion by ArtemLiA** [etna-team/etna#485] [Q&A] [answered]
Hi there! I would like to ask if there are any plans to add support of Python3.11 and Python3.12?

**Comment by d-a-bunin** [comment, marked as answer]
Hi! There is a related discussion #420.
We are currently blocked by using old version of `pytorch_forecasting`. In a next major release we are planning to remove this dependency and update python.

As for the timing of the next major release, we haven't made a particular decision about it yet, because it is difficult to select other things we want to change. Probably, we'll just update dependencies and make some small break changes in API.

**Comment by egoriyaa** [comment]
Added 3.11 and 3.12 versions in #599
